colcon_ros_gradle/task/ament_gradle/build.py

```diff
@@ -23,6 +25,21 @@ def add_arguments(self, *, parser):  # noqa: D102
             '--ament-gradle-task',
             help='Run a specific task instead of the default task')
 
+    async def build(
+        self, *, additional_hooks=[], skip_hook_creation=False
+    ):  # noqa: D102
+        pkg = self.context.pkg
+        args = self.context.args
+
+        # TODO(jacobperron): This would be better handled by a more generic ament_java library
+        additional_hooks += create_environment_hook(
+            'ament_gradle_prefix_path', Path(args.install_base), pkg.name,
```

Comment:
Shouldn't this be 'ament_prefix_path'? That seems what the build task for `ament_python` build types do.

Reply:
I mean this snippet in `ament_python`:

https://github.com/colcon/colcon-ros/blob/master/colcon_ros/task/ament_python/build.py#L40-L42

Reply:
This is the name of the hooks script that gets generated. I think I chose the name "ament_gradle_prefix_path" to avoid possible collisions with existing hooks, but maybe it's not necessary. There's probably no harm is renaming this "ament_prefix_path" if you prefer.

Reply:
Renamed the hook to "ament_prefix_path" (3c4a088)